core/src/main/java/com/netflix/conductor/core/execution/evaluators/JavascriptEvaluator.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright 2021 Netflix, Inc.
+ *  <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.core.execution.evaluators;
+
+import java.util.Map;
+
+import javax.script.ScriptException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.netflix.conductor.core.events.ScriptEvaluator;
+import com.netflix.conductor.core.exception.TerminateWorkflowException;
+
+public class JavascriptEvaluator implements Evaluator {
+
+   public static final String NAME = "javascript";
+   private static final Logger LOGGER = LoggerFactory.getLogger(JavascriptEvaluator.class);
+
+   @Override
+   public Object evaluate(String expression, Object input) {
+      LOGGER.debug("Javascript evaluator -- expression: {}", expression);
+      try {
+         //Evaluate the expression by using the Nashhorn based script evaluator
```

Comment:
Since we will be upgrading from Nashorn to GraalVM soon and also the fact that this implementation is not tied to the js engine, the comment can omit this detail.

Reply:
Fixed